Source/Scene/GroundPolylineBatch.js

```diff
@@ -0,0 +1,501 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/Color',
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/IndexDatatype',
+        '../Core/Matrix4',
+        '../Core/TranslationRotationScale',
+        '../Renderer/Buffer',
+        '../Renderer/BufferUsage',
+        '../Renderer/DrawCommand',
+        '../Renderer/RenderState',
+        '../Renderer/ShaderProgram',
+        '../Renderer/ShaderSource',
+        '../Renderer/VertexArray',
+        '../Shaders/GroundPolylineBatchVS',
+        '../Shaders/PolylineCommon',
+        './BlendingState',
+        './Pass'
+    ], function(
+        Cartesian3,
+        Color,
+        ComponentDatatype,
+        defined,
+        destroyObject,
+        IndexDatatype,
+        Matrix4,
+        TranslationRotationScale,
+        Buffer,
+        BufferUsage,
+        DrawCommand,
+        RenderState,
+        ShaderProgram,
+        ShaderSource,
+        VertexArray,
+        GroundPolylineBatchVS,
+        PolylineCommon,
+        BlendingState,
+        Pass) {
+    'use strict';
+
+    /**
+     * Renders a batch of polylines that have been subdivided to be draped on terrain.
+     *
+     * @alias GroundPolylineBatch
+     * @constructor
+     * @private
```

Comment:
This should go at the bottom of the reference doc: https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/DocumentationGuide#private

At one point it might have even been required to be at the bottom, but that could have been at jsdoc-toolkit bug.